### PR TITLE
Fix #2209: Submission re-run functionality.

### LINKED ISSUE #2209: Add a feature to re-reun a submission by challenge host using UI
## Current Behaviour
If a challenge host wants to re-run a submission then the host has to email EvalAI admin to re-run a particular submission from admin.

## Expected Behaviour
The challenge host should be able to re-run the submission from the UI.

Please feel free to post the approach first.

### PR DESCRIPTION
**Pull request by yashdusing**
Fix #2209 : Adds the functionality to re-run submissions from 'all submissions' page.

Steps to verify : 
- 1. From the admin panel, change the status of any of the submission to cancelled.
- 2. Load the 'all submissions' page, which will have have the cancelled submission and re-run the submission from the button
- 3. Refresh the page and load all submissions again (currently trying to fix that by dynamically refreshing submissions when re-run button is clicked)

### CODE REVIEW

**Comment by yashdusing** [comment]
@deshraj @RishabhJain2018 Please review the PR

**Comment by yashdusing** [comment]
@RishabhJain2018 please review

**Comment by yashdusing** [comment]
@RishabhJain2018 Please review !

File: apps/jobs/urls.py [hunk 1]
```diff
@@ -20,6 +20,11 @@
         views.get_remaining_submissions,
         name="get_remaining_submissions",
     ),
+    url(
+        r'^challenge/(?P<challenge_id>[0-9]+)/'
```
Comment (RishabhJain2018) [inline apps/jobs/urls.py]: Please remove challenge_id and challenge_phase_id from the URL. I think submission ID is sufficient as a submission is related to a phase and phase is related to a challenge in the models.

**Comment by RishabhJain2018** [comment]
@yashdusing Are you working on this?

**Comment by yashdusing** [comment]
@RishabhJain2018 yes. Will start work tomorrow

**Comment by yashdusing** [comment]
@RishabhJain2018 made the changes. Please review

**Pull request closed without merging**